src/engine/Server.cpp

```diff
@@ -380,33 +380,31 @@ boost::asio::awaitable<void> Server::processQuery(
           request));
     }
 
+    const auto throwIfConstructClause = [&pq]() {
+      if (pq.hasConstructClause()) {
+        throw std::runtime_error{
+            "CONSTRUCT queries only support turtle syntax right now"};
```

Comment:
Two remarks remaining here:

1. You should add the turtle media type in the `supportedMediaTypes` above (line 345) s.t. we can also specify it via an accept header.
2.  replace "turtle syntax" by "RDF Turtle as an export format". And Add "Please specify "action=turtle-export" as a query parameter or the accept header "...(add corresponding header here)

Reply:
It's currently called `turtle_export` to be consistent with `csv_export` and `tsv_export` and all the other options